newrelic/nri-kubernetes
# Add cronjob to testing resources.

The Kubernetes team is working on adding new metrics for all Kubernetes workloads, including `cronjob`. As a first step in that process, this PR adds a `cronjob` workload that will be used in the local dev environment. 

### Testing Plan

1. Run local dev environment: `make local-env-start`
2. Observe `cronjob` running every 1 minute
